setup.py

```diff
@@ -50,8 +31,26 @@ def get_requirements():
     "sequential program and makes recommendations on how to exploit it.",
     # long_description=open(SRC / "README.md").read(),
     long_description_content_type="text/markdown",
-    install_requires=get_requirements(),
-    extras_require={"dev": ["mypy", "black", "pre-commit"]},
+    install_requires=[
+        "contextlib2>=0.5.5",
+        "docopt>=0.6.2",
+        "extrap",
+        "jsonpickle",
+        "jsons",
+        "lxml>=4.3.3",
+        "matplotlib",
+        "networkx",
+        "numpy>=1.16.3",
+        "pluginbase>=1.0.0",
+        "pstats2",
+        "schema>=0.7.0",
+        "sympy",
+        "sympy_plot_backends",
+    ],
```

Comment:
I would consider moving those into a separate file for easier maintainability, but the current state is fine with me as well.